uplink/converters/__init__.py

```diff
@@ -13,14 +13,16 @@
 # fmt: off
 from uplink.converters.standard import StandardConverter
 from uplink.converters.marshmallow_ import MarshmallowConverter
-from uplink.converters.pydantic_ import PydanticConverter
+from uplink.converters.pydantic_v1 import PydanticV1Converter
+from uplink.converters.pydantic_v2 import PydanticV2Converter
 from uplink.converters.typing_ import TypingConverter
 # fmt: on
 
 __all__ = [
     "StandardConverter",
     "MarshmallowConverter",
-    "PydanticConverter",
```

Comment:
Removing this might be a breaking change, since it is being exported by the `uplink.converters` package.